packages/typespec-ts/src/transform/transfromRLCOptions.ts

```diff
@@ -218,17 +219,43 @@ function detectIfNameConflicts(dpgContext: SdkContext) {
   return false;
 }
 
-function getIncludeShortcuts(emitterOptions: RLCOptions) {
+function getIncludeShortcuts(emitterOptions: EmitterOptions) {
   return Boolean(emitterOptions.includeShortcuts);
 }
 
-function getBranded(emitterOptions: RLCOptions) {
-  return emitterOptions.branded !== undefined ? emitterOptions.branded : true;
+function getFlavor(
+  emitterOptions: EmitterOptions,
+  packageDetails?: PackageDetails
+): PackageFlavor | undefined {
```

Comment:
nit: I think this can also just return PackageFlavor, as it is `"azure" | undefined`